cloudinit/serial.py

```diff
@@ -1,12 +1,16 @@
 # This file is part of cloud-init. See LICENSE file for license information.
 
+from typing import Type
+
 try:
-    from serial import Serial
+    from serial import Serial as _Serial
```

Comment:
Looks like the try/except is from the 2.6 days. If this is still necessary, could we update the comments explaining why (in terms of python3)?

Reply:
Thanks for pointing out this @brettmilford. I have removed it and everything works.

Reply:
Thanks!